src/theme/foundations/global.ts

```diff
@@ -38,7 +38,7 @@ const global = {
       _focus: activeFocus(),
     },
   },
-  "iframe, [href], [tabindex], [contentEditable='true']": {
```

Comment:
This is probably a question for @EdwinGuzman and @bigfishdesign13, but how do we know what the impact of this change is outside of the bug that Isa's resolving?

Reply:
And, out of curiosity, is this issue resolvable on the Scout side rather than in DS? Not saying it shouldn't be changed in the DS, but if we are unclear on the overall impact, is it an option?

Reply:
With the removal of the `[tabindex]` selector, what will happen when focus is placed on a text element? For example, after using pagination on a list of search results, we often scroll to the top of the list and move focus to either a text element that describes the updated results list or focus is placed on the first item in the list.

Reply:
@oliviawongnyc and I were chatting a bit about this the other day. I do agree that using an attribute selector is not best practice. If we really need to, we should instead consider creating a new rule block just for iframe and tabIndex that uses a similar set of focus styles, but doesn't include the z-index rule that is giving us trouble.

I also tested DS components and programmatically focusing on other elements still works. This is true for `NewsletterSignup` and `FeedbackBox`, although `FeedbackBox` was not always consistent.

> what will happen when focus is placed on a text element? 

This should continue to work, but let's further test in turbine.